stock_operating_unit/tests/test_stock_picking.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 ForgeFlow S.L.
+# Copyright 2019 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from . import test_stock_operating_unit as test_stock_ou
+
+
+class TestStockPicking(test_stock_ou.TestStockOperatingUnit):
```

Comment:
```suggestion
from odoo.tests import tagged

from . import test_stock_operating_unit as test_stock_ou


@tagged("post_install", "-at_install")
class TestStockPicking(test_stock_ou.TestStockOperatingUnit):
```
I suggest including the decorator "tagged" with the tags "post_install" and taking out the tag "at_install" to avoid IntegrityError such as _null value in column "sale_line_warn" of relation "product_template" violates not-null constraint._